Mathlib/CategoryTheory/Functor/KanExtension/Pointwise.lean

```diff
@@ -55,6 +56,108 @@ abbrev HasPointwiseRightKanExtensionAt (Y : D) :=
 that it has a pointwise right Kan extension at any object. -/
 abbrev HasPointwiseRightKanExtension := ∀ (Y : D), HasPointwiseRightKanExtensionAt L F Y
 
+lemma hasPointwiseLeftKanExtensionAt_iff_of_iso {Y₁ Y₂ : D} (e : Y₁ ≅ Y₂) :
+    HasPointwiseLeftKanExtensionAt L F Y₁ ↔
+      HasPointwiseLeftKanExtensionAt L F Y₂ := by
+  revert Y₁ Y₂ e
+  suffices ∀ ⦃Y₁ Y₂ : D⦄ (_ : Y₁ ≅ Y₂) [HasPointwiseLeftKanExtensionAt L F Y₁],
+      HasPointwiseLeftKanExtensionAt L F Y₂ from
+    fun Y₁ Y₂ e => ⟨fun _ => this e, fun _ => this e.symm⟩
+  intro Y₁ Y₂ e _
+  change HasColimit ((CostructuredArrow.mapIso e.symm).functor ⋙ CostructuredArrow.proj L Y₁ ⋙ F)
+  infer_instance
+
+lemma hasPointwiseRightKanExtensionAt_iff_of_iso {Y₁ Y₂ : D} (e : Y₁ ≅ Y₂) :
+    HasPointwiseRightKanExtensionAt L F Y₁ ↔
+      HasPointwiseRightKanExtensionAt L F Y₂ := by
+  revert Y₁ Y₂ e
+  suffices ∀ ⦃Y₁ Y₂ : D⦄ (_ : Y₁ ≅ Y₂) [HasPointwiseRightKanExtensionAt L F Y₁],
+      HasPointwiseRightKanExtensionAt L F Y₂ from
+    fun Y₁ Y₂ e => ⟨fun _ => this e, fun _ => this e.symm⟩
+  intro Y₁ Y₂ e _
+  change HasLimit ((StructuredArrow.mapIso e.symm).functor ⋙ StructuredArrow.proj Y₁ L ⋙ F)
+  infer_instance
+
+variable {L} in
+/-- `HasPointwiseLeftKanExtensionAt` is invariant when we replace `L` by an equivalent functor. -/
+lemma hasPointwiseLeftKanExtensionAt_iff_of_iso' {L' : C ⥤ D} (e : L ≅ L') (Y : D) :
```

Comment:
```suggestion
variable {L} in
/-- `HasPointwiseLeftKanExtensionAt` is invariant when we replace `L` by an isomorphic functor. -/
lemma hasPointwiseLeftKanExtensionAt_iff_of_natIso {L' : C ⥤ D} (e : L ≅ L') (Y : D) :
```

Reply:
In the category theory API, lemmas involving isomorphisms of functors use either `iso` or `natIso`. If we had to "unify" the names in the category theory API, I would rather replace `_natIso` by `_iso`, so that I prefer sticking with these names.

Reply:
In this case though, wouldn't you agree it gives a clearer distinction from the lemma which has the unprimed version of the same name?

Reply:
I just noticed that I had the very same suggestion pending. I agree with Dagur here.